src/audio/module_adapter/iadk/iadk_module_adapter.cpp

```diff
@@ -37,45 +37,55 @@ int IadkModuleAdapter::IadkModuleAdapter_Prepare(void)
 	return 0;
 }
 
-uint32_t IadkModuleAdapter::IadkModuleAdapter_Process(struct input_stream_buffer *input_buffers,
-						      int num_input_buffers,
-						      struct output_stream_buffer *output_buffers,
-						      int num_output_buffers)
+uint32_t IadkModuleAdapter::IadkModuleAdapter_Process(struct sof_source **sources,
```

Comment:
please add sof_source sof_sink related header file into source header:
There should have: #include "#include <sof/audio/sink_api.h> etc.

Reply:
There is no need of adding sink_api.h because it is included in module_interface.h and it is unnecessarily redundant.

Reply:
you need add all the headers in source file explicitly for current source, this is a coding standard, I followed in my refactor code change.

Reply:
Is it in some linux coding standard? I would like to learn more about it.

Reply:
fixed

Reply:
headers added.